tests/loginValidUser.test.js

```diff
@@ -0,0 +1,21 @@
+const { test, expect } = require('@playwright/test');
+const { LoginPage } = require('../pages/loginPage.js');
+const { HomePage } = require('../pages/homePage.js');
+
+
+
+test.describe('Redmine login', () => {
+    test('should login with existing user', async ({ page }) => {
+        const homePage = new HomePage(page);
+        const loginPage = new LoginPage(page);
+
+        await loginPage.goto('https://www.redmine.org/login');
```

Comment:
@karenn32 This action must be in the 'beforeEach' hook (or 'beforeAll' in case, if the test spec contains only 1 test case.).